components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -2539,15 +2759,15 @@ private Group addDomainToUserMembers(Group group, String userStoreDomain) throws
     }
 
     /**
-     * check whether the filtering is supported
-     * @param filterOperation operator use for filtering
+     * Method to check whether the filtering is supported.
+     *
+     * @param filterOperation Operator to be used for filtering
      * @return boolean to check whether operator is supported
      */
     private boolean isNotFilteringSupported(String filterOperation) {
```

Comment:
Shall we change the name to `isFilteringNotSupported`?

Reply:
Fixed: 0811c8c408e0d35596e71d82f167868b3f079ae7